src/main/java/org/cascadebot/cascadebot/commands/developer/DumpCommand.java

```diff
@@ -62,7 +62,7 @@ public String command() {
 
     @Override
     public String description() {
-        return "dump command";
+        return "Dump command";
```

Comment:
This really isn't more descriptive 😉

Reply:
i still don't understand what the dump command does so eh